modules/compute/vm-instance/variables.tf

```diff
@@ -203,6 +205,11 @@ variable "network_interfaces" {
   }
 }
 
+variable "region" {
```

Comment:
nit: We could compute this from the zone by hacking off the end. It might be more robust to pass it so I will let you make the call.

Reply:
I considered it originally and, while this is a breaking change, I think most blueprints already receive both region and zone. I would prefer not to create a precedent having modules compute zone automatically.